ntoskrnl/io/pnpmgr/devaction.c

```diff
@@ -248,12 +248,12 @@ IopCreateDeviceInstancePath(
 
     DPRINT("Sending IRP_MN_QUERY_CAPABILITIES to device stack (after enumeration)\n");
 
-    Status = IopQueryDeviceCapabilities(DeviceNode, &DeviceCapabilities);
+    Status = PiIrpQueryPnPDeviceCapabilities(DeviceNode, &DeviceCapabilities);
     if (!NT_SUCCESS(Status))
     {
         if (Status != STATUS_NOT_SUPPORTED)
         {
-            DPRINT1("IopQueryDeviceCapabilities() failed (Status 0x%08lx)\n", Status);
+            DPRINT1("PiIrpQueryPnPDeviceCapabilities() failed (Status 0x%08lx)\n", Status);
         }
```

Comment:
Please, just remove this DPRINT. I'm going to implement some kind of central logging inside pnpirp.c